.github/workflows/build.yml

```diff
@@ -200,73 +200,73 @@ jobs:
           extra_build_args: |
             GOLANG_VERSION=${{ needs.get-go-version.outputs.go-version }}
 
-  build-docker-redhat:
-    name: Docker UBI Image Build (for Red Hat Certified Container Registry)
-    needs:
-      - get-go-version
-      - set-product-version
-      - build-linux
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        include:
-          - { target-name: "release-ubi", registry-id: "631f805e0d15f623c5996c2e" }
-    env:
-      repo: ${{github.event.repository.name}}
-      version: ${{needs.set-product-version.outputs.product-version}}
-    steps:
-      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
-      - uses: hashicorp/actions-docker-build@76d2fc91532d816ca2660d8f3139e432ac3700fd
-        with:
-          version: ${{env.version}}
-          target: ${{ matrix.target-name }}
-          arch: amd64
-          redhat_tag: quay.io/redhat-isv-containers/${{matrix.registry-id}}:${{env.version}}-ubi
+  # build-docker-redhat:
+  #   name: Docker UBI Image Build (for Red Hat Certified Container Registry)
+  #   needs:
+  #     - get-go-version
+  #     - set-product-version
+  #     - build-linux
+  #   runs-on: ubuntu-latest
+  #   strategy:
+  #     matrix:
+  #       include:
+  #         - { target-name: "release-ubi", registry-id: "631f805e0d15f623c5996c2e" }
+  #   env:
+  #     repo: ${{github.event.repository.name}}
+  #     version: ${{needs.set-product-version.outputs.product-version}}
+  #   steps:
+  #     - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+  #     - uses: hashicorp/actions-docker-build@76d2fc91532d816ca2660d8f3139e432ac3700fd
+  #       with:
+  #         version: ${{env.version}}
+  #         target: ${{ matrix.target-name }}
+  #         arch: amd64
+  #         redhat_tag: quay.io/redhat-isv-containers/${{matrix.registry-id}}:${{env.version}}-ubi
```

Comment:
Removing redhat builds since I'm not sure rel-eng set this up. I don't think we need this for the release since there was no new requirement for it.